packages/component/src/Activity/ActivityFeedback.tsx

```diff
@@ -59,9 +62,16 @@ function ActivityFeedback({ activity }: ActivityFeedbackProps) {
 
   const feedbackActions = useMemo<ReadonlySet<OrgSchemaAction>>(() => {
```

Comment:
Let's treat the `feedbackActions` as source of truth. Try this logic out and see if it would simplify things and reduce SoT:

`feedbackSubmitted` means: any `LikeAction`/DislikeAction` in the `potentialAction` has status of `CompletedActionStatus`.

Try to drive all the UIs based on the content of `potentialAction`. Says, if the `LikeAction.actionStatus === 'CompletedActionStatus'`, it cannot be reset (because it is submitted.)

Reply:
@lexi-taylor this means we don't need the state such as:
```ts
  const [feedbackSubmitted, setFeedbackSubmitted] = useState<boolean>(false);
```

We can just derrive it from the submittedAction, as follows:
```ts
  const feedbackSubmitted = submittedAction?.actionStatus === 'CompletedActionStatus'
```

And so on.